planner/core/testdata/plan_suite_out.json

```diff
@@ -1115,22 +1115,22 @@
       {
         "SQL": "select /*+ HASH_JOIN(@sel_1 t1), SM_JOIN(@sel_2 t2) */ t1.a, t1.b from t t1, (select t2.a from t t2, t t3 where t2.a = t3.c) s where t1.a=s.a",
         "Plan": "RightHashJoin{TableReader(Table(t))->MergeInnerJoin{TableReader(Table(t))->IndexReader(Index(t.c_d_e)[[NULL,+inf]])}(Column#13,Column#27)}(Column#1,Column#13)->Projection",
-        "Hints": "USE_INDEX(@`sel_1` `t1` ), USE_INDEX(@`sel_2` `t2` ), USE_INDEX(@`sel_2` `t3` `c_d_e`), SM_JOIN(@`sel_2` `t2`, `t3`), HASH_JOIN(@`sel_1` `t1`)"
+        "Hints": "USE_INDEX(@`sel_1` `t1` ), USE_INDEX(@`sel_2` `t2` ), USE_INDEX(@`sel_2` `t3` `c_d_e`), SM_JOIN(@`sel_2` `t2`), HASH_JOIN(@`sel_1` `t1`)"
       },
       {
         "SQL": "select /*+ HASH_JOIN(@sel_1 t1), SM_JOIN(@qb t2) */ t1.a, t1.b from t t1, (select /*+ QB_NAME(qb) */ t2.a from t t2, t t3 where t2.a = t3.c) s where t1.a=s.a",
         "Plan": "RightHashJoin{TableReader(Table(t))->MergeInnerJoin{TableReader(Table(t))->IndexReader(Index(t.c_d_e)[[NULL,+inf]])}(Column#13,Column#27)}(Column#1,Column#13)->Projection",
-        "Hints": "USE_INDEX(@`sel_1` `t1` ), USE_INDEX(@`sel_2` `t2` ), USE_INDEX(@`sel_2` `t3` `c_d_e`), SM_JOIN(@`sel_2` `t2`, `t3`), HASH_JOIN(@`sel_1` `t1`)"
```

Comment:
We don't need to figure out all the tables which use merge join?

Reply:
One table is enough. If it's not stable, it's caused by join order instead of the join algorithm hint.